standard_plus_issue/views/standard_plus_issue.xml

```diff
@@ -113,5 +113,11 @@
               parent="menu_standard_plus_issue"
               action="action_standard_plus_issue"/>
 
```

Comment:
Shouldn't `menuitem id="menu_standard_plus_issue"`and `menuitem id="menu_standard_plus_issue_sub"` removed?